app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenControllerTest.kt

```diff
@@ -564,7 +559,6 @@ class PatientEntryScreenControllerTest {
   }
 
   private fun screenCreated() {
```

Comment:
This function can be inlined since the only thing it does is call `screenCreatedForMobius()`.